src/main/java/com/microsoft/graph/Requests/BaseClient.java

```diff
@@ -1,6 +1,5 @@
 package com.microsoft.graph.Requests;
 
-import com.microsoft.graph.content.BatchRequestBuilder;
 import com.microsoft.kiota.RequestAdapter;
```

Comment:
I believe we should drop that class all together

Reply:
Keeping BaseClient.java to keep in line with the DotNet library

Reply:
sure. I don't think it's used anywhere since it's internal, maybe we shuold switch it to package instead of public to align as well?

Reply:
re-opened this one since we didn't reach a resolution

Reply:
class now uses the default modifier, ie no modifier, so that it accessible only by classes within same package.